src/quoridor/server/server.go

```diff
@@ -66,14 +70,27 @@ func getGameHandler(w http.ResponseWriter, r *http.Request) {
 	response.SendOK(w, game)
 }
 
+func joinGameHandler(w http.ResponseWriter, r *http.Request) {
+	id := request.GetGameID(r)
+	gameToken:= shortuuid.New()
+	game, err := gamecontroller.JoinGame(id, gameToken)
+	if err != nil {
+		response.SendBadRequestError(w, err)
+		return
+	}
+	w.Header().Set("Quoridor-Token", gameToken)
+	response.SendOK(w, game)
+}
+
 func addFenceHandler(w http.ResponseWriter, r *http.Request) {
 	id := request.GetGameID(r)
 	fence, err := request.GetFence(r)
 	if err != nil {
 		response.SendBadRequestError(w, err)
 		return
 	}
-	game, err := gamecontroller.AddFence(id, fence)
+	gameToken := r.Header.Get("Quoridor-Token")
```

Comment:
Moreover, it's not a "gameToken" but an "userToken" or "authorizationToken" or "sessionToken" ...